robottelo/api/utils.py

```diff
@@ -37,3 +38,31 @@ def enable_rhrepo_and_fetchid(basearch, org_id, product, repo,
             result = entities.Repository(name=repo).search(
                 query={'organization_id': org_id})
     return result[0].id
+
+
+def promote(content_view_version, environment_id):
+    """Call ``content_view_version.promote(…)``.
+
+    :param content_view_version: A ``nailgun.entities.ContentViewVersion``
+        object.
+    :param environment_id: An environment ID.
+    :returns: Whatever ``nailgun.entities.ContentViewVersion.promote`` returns.
+
+    """
+    return content_view_version.promote(data={
+        u'environment_id': environment_id
+    })
+
+
+def upload(organization_id, manifest):
```

Comment:
Since one can upload packages (rpms, puppet, eventually docker) artifacts into a Satellite, should we name this `upload_manifest` instead?